src/timer_esp32.rs

```diff
@@ -21,8 +21,7 @@ const TIMER_DELAY: fugit::MicrosDurationU64 = fugit::MicrosDurationU64::millis(1
 #[cfg(not(debug_assertions))]
 const TIMER_DELAY: fugit::MicrosDurationU64 = fugit::MicrosDurationU64::millis(3);
 
-static mut TIMER1: SpinLockMutex<RefCell<Option<Timer<Timer0<TIMG1>>>>> =
-    SpinLockMutex::new(RefCell::new(None));
+static mut TIMER1: Mutex<RefCell<Option<Timer<Timer0<TIMG1>>>>> = Mutex::new(RefCell::new(None));
```

Comment:
I believe both these can just be `static`

Reply:
oh yes - I always forget about that 👍